tests/sentry/sentry_metrics/test_indexer_cache.py

```diff
@@ -17,39 +17,49 @@
 
 @pytest.fixture
 def use_case_id() -> str:
-    return UseCaseKey.RELEASE_HEALTH.value
+    return UseCaseID.SESSIONS.value
 
 
 def test_cache(use_case_id: str) -> None:
     cache.clear()
-    assert indexer_cache.get("blah", use_case_id) is None
-    indexer_cache.set("blah", 1, use_case_id)
-    assert indexer_cache.get("blah", use_case_id) == 1
+    assert indexer_cache.get(f"{use_case_id}:1:blah:123") is None
+    indexer_cache.set(f"{use_case_id}:1:blah:123", 1)
+    assert indexer_cache.get(f"{use_case_id}:1:blah:123") == 1
 
-    indexer_cache.delete("blah", use_case_id)
-    assert indexer_cache.get("blah", use_case_id) is None
+    indexer_cache.delete(f"{use_case_id}:1:blah:123")
+    assert indexer_cache.get(f"{use_case_id}:1:blah:123") is None
 
 
 def test_cache_many(use_case_id: str) -> None:
     cache.clear()
-    values = {"hello": 2, "bye": 3}
-    assert indexer_cache.get_many(list(values.keys()), use_case_id) == {"hello": None, "bye": None}
-    indexer_cache.set_many(values, use_case_id)
-    assert indexer_cache.get_many(list(values.keys()), use_case_id) == values
+    values = {f"{use_case_id}:100:hello": 2, f"{use_case_id}:100:bye": 3}
+    assert indexer_cache.get_many(values.keys()) == {
+        f"{use_case_id}:100:hello": None,
+        f"{use_case_id}:100:bye": None,
+    }
+    indexer_cache.set_many(values)
+    assert indexer_cache.get_many(list(values.keys())) == values
 
-    indexer_cache.delete_many(list(values.keys()), use_case_id)
-    assert indexer_cache.get_many(list(values.keys()), use_case_id) == {"hello": None, "bye": None}
+    indexer_cache.delete_many(list(values.keys()))
+    assert indexer_cache.get_many(values.keys()) == {
+        f"{use_case_id}:100:hello": None,
+        f"{use_case_id}:100:bye": None,
+    }
 
 
 def test_make_cache_key(use_case_id: str) -> None:
-    key = indexer_cache.make_cache_key("blah", "release-health")
-    assert key == f"indexer:test:org:str:{use_case_id}:{md5_text('blah').hexdigest()}"
+    orgId = 1
+    string = ":blah:blah"
+    nameSpace = md5_text(f"{orgId}:{string}").hexdigest()
+    key = indexer_cache.make_cache_key(f"{use_case_id}:{orgId}:{string}")
+
+    assert key == f"indexer:test:org:str:{use_case_id}:{nameSpace}"
 
 
 def test_formatted_results(use_case_id: str) -> None:
-    values = {"hello": 2, "bye": 3}
-    results = {indexer_cache.make_cache_key(k, use_case_id): v for k, v in values.items()}
-    assert indexer_cache._format_results(list(values.keys()), results, use_case_id) == values
+    values = {f"{use_case_id}::hello": 2, f"{use_case_id}::bye": 3}
```

Comment:
why is there a double colon `::` here? is there supposed to be an org id in the string?

Reply:
Just to see if the extra delimiter character will mess things up